src/qt/mainwindow.cpp

```diff
@@ -79,11 +81,12 @@ MainWindow::MainWindow(QWidget *parent) :
 
     ui->widget_contentFilter->setVisible(false);
 
-    this->m_shapesDemo.init();
-    if(m_shapesDemo.isInitialized())
-    {
-        addMessageToOutput(QString("RTPSParticipant ready in domainId %1").arg(m_shapesDemo.getOptions().m_domainId),true);
-    }
+    // Initialize when a new Endpoint is added
+    // this->m_shapesDemo.init();
+    // if(m_shapesDemo.isInitialized())
+    // {
+    //     addMessageToOutput(QString("RTPSParticipant ready in domainId %1").arg(m_shapesDemo.getOptions().m_domainId),true);
+    // }
```

Comment:
Can we remove this code or is another pending `TODO`?

Reply:
:+1: